test/jdk/jni/nullCaller/exeNullCallerTest.cpp

```diff
@@ -184,3 +187,22 @@ int main(int argc, char** args) {
     return 0;
 }
 
+int main(int argc, char *argv[]){
+#ifdef AIX
+   size_t adjusted_stack_size = 1024*1024;
+   pthread_t id;
+   int result;
+   pthread_attr_t attr;
+   pthread_attr_init(&attr);
+   pthread_attr_setstacksize(&attr, adjusted_stack_size);
+   result = pthread_create(&id, &attr, run, (void *)argv);
+   if (result != 0) {
+     fprintf(stderr, "Error: pthread_create failed with error code %d \n", result);
+     return -1;
+   }
+   pthread_join(id, NULL);
+#else
+   run(&argv);
+#endif //AIX
```

Comment:
The tests in test/jdk use 4 space indent rather than 2. Otherwise the changes to these tests look okay, just a bit annoying to have the same #ifdef AIX ... #endif code copied into each test.

Reply:
Thank you @AlanBateman, I will update with the indentation fixes. [12006](https://github.com/openjdk/jdk/pull/12006) was a parent issue. I thought to make a PR [12302](https://github.com/openjdk/jdk/pull/12302) for other tests after the changes for PR [12006](https://github.com/openjdk/jdk/pull/12006) is accepted by the community.